### PR TITLE
191 replace faq w sponsors

### PR DESCRIPTION
**Pull request by hakuna-matatas**
Replaced FAQ with trevor's summary paragraph. Sponsor section was unchanged.

### CODE REVIEW

File: views/index.jade [hunk 1]
```diff
@@ -158,18 +158,15 @@ block append content
...
                         p closing ceremony
                         h5 call auditorium
+            br
+            br
+            br
+            br
+            br
+            br
+            br
+            br
```
Comment (julianahhong) [inline views/index.jade]: could we adjust spacing using margins instead of breaks? using css (less) so it looks cleaner

File: views/index.jade [hunk 2]
```diff
@@ -180,80 +177,38 @@ block append content
...
-                        .redfish(style="top:75%;" data-900-top="left:70%;" data--200-top="left:-10%;")
-                        .bluefish(style="top:78%;" data-900-top="left:90%;" data--200-top="left:5%;")
-                        .orangefish(style="top:82%;" data-900-top="left:85%;" data--200-top="left:7%;")
-                        .redfish(style="top:85%;" data-900-top="left:85%;" data--200-top="left:2%;")
+                    br
+                    p
+                      | BigRed//Hacks is the oldest student-run, large-scale hackathon at Cornell University. Every semester, the BigRed//Hacks team hosts a three day
+                      | hackathon event, inviting students from all across the United States, from a multitude of colleges and universities, to come together to build the coolest
+                      | and most innovative projects, as well as meet new stduents, talk to people who are actually in the work force, and win prizes along the way.
+                    p
```
Comment (julianahhong) [inline views/index.jade]: students typo

File: views/index.jade [hunk 3]
```diff
@@ -180,80 +177,38 @@ block append content
...
+                      | This year, in the fall of 2016, we had over 210 teams participate in the BigRed/Hacks tournament, consisting of students from a wide variety of universities.
+                      | Awards were divided into 6 main prizes and 6 special prizes. For the main prizes, Best UI Design was awarded to Recycle Rampage, Best Business Project went to Add.me,
+                      | Best Technical Feat went to Andreessen, Best Sustainability Hack was rewareded to BinGO, Best First Time Hack went to PuppetPlant, and Best Hardware Hack went to JankTank.
+                      | For the special prizes, Best Use of Nessie API went to Cheddar and GarbagemonGO, Best Use of Microsoft Technology went to BinGO, Best Cornell-Related Hack went to
+                      | CUCA (Cornell Course Advisor), Most Security Conscious App and Best Data Privacy Hack went to HTTPS Security Demo, and #HackHarassment went to Kindernet.
+                    p
```
Comment (julianahhong) [inline views/index.jade]: this section is unnecessary since @hsun1316 is making a section with the winners

File: views/index.jade [hunk 4]
```diff
@@ -180,80 +177,38 @@ block append content
...
+                      | We try to create a theme for each hackathon event. The Fall 2016 hackathon was all about sustainability: we wanted to get people more interested in trying to develop projects
+                      | that could be applied to preserving the environment, proper waste removal, conservation of water, amongst other things. We hope that in the future, more and more hacks will be
+                      | created for the purpose of protecting our planet Earth.
+
```
Comment (julianahhong) [inline views/index.jade]: I'm gonna rewrite this a bit

File: public/css/index.less [hunk 5]
```diff
@@ -484,16 +484,29 @@ section {
...
+  padding-bottom: 180px;
+  opacity: 0.8;
 
+  h2, dd, dt {
+    color: white;
+  }
+  .container {
+    color: @off-white;
+  }
+  background: @ocean-blue;
```
Comment (mahsu) [inline public/css/index.less]: group this with the other `.sustainable` properties.

File: public/css/index.less [hunk 6]
```diff
@@ -484,16 +484,29 @@ section {
...
+  dt {
+    font-size: 20px;
+    padding-top: 10px;
+  }
+  dd{
```
Comment (mahsu) [inline public/css/index.less]: `dd {`

File: public/css/index.less [hunk 7]
```diff
@@ -484,16 +484,29 @@ section {
...
+    font-size: 16px;
+    font-weight: @weight-normal;
+  }
+  a{
```
Comment (mahsu) [inline public/css/index.less]: `a {`

File: views/index.jade (hunk 1 above)
Comment (mahsu) [inline views/index.jade]: +1 using margins also makes it easier to maintain compatibility across different devices.

File: public/css/index.less [hunk 8]
```diff
@@ -245,7 +245,7 @@ section {
   left: 30%;
 }
 
-.buttons{
+.buttons {
   //position: relative;
```
Comment (mahsu) [inline public/css/index.less]: remove

**Comment by TrevorEdwards** [comment]
I merged master and edited a bit to fix a small margin issue. I'm fine with it as soon as @julianahhong updates the content of the paragraph

File: views/index.jade (hunk 4 above)
Comment (julianahhong) [inline views/index.jade]: "This year’s fall hackathon was focused on sustainability. We wanted to encourage critical thinking and innovative problem-solving related to issues of global importance, and we were not disappointed. Over three days, our hackers developed creative solutions in the form of games, apps, and websites to tackle challenges facing our planet Earth." Perhaps something like this.. @TrevorEdwards or @mahsu  feel free to edit

Comment (TrevorEdwards) [inline views/index.jade]: looks good but remove "Earth" i think

**Comment by julianahhong** [comment]
@TrevorEdwards looks good!

**Pull request merged**